src/main/core/Entity/Facet/FieldFacet.php

```diff
@@ -157,6 +157,27 @@ class FieldFacet
      */
     protected $help;
 
+    /**
+     * @ORM\Column(name="dependency_field")
+     *
+     * @var string
+     */
+    protected $dependencyField;
+
+    /**
+     * @ORM\Column(name="validation_type")
+     *
+     * @var string
+     */
+    protected $validationType;
+
+    /**
+     * @ORM\Column(name="comparison_value")
+     *
+     * @var string
+     */
+    protected $comparisonValue;
```

Comment:
Should be a `json` type to be able to allow all field types (not only choice and boolean)